dev/io.openliberty.checkpoint_fat/publish/servers/checkpointFaces-beanServer/bootstrap.properties

```diff
@@ -13,4 +13,4 @@
 bootstrap.include=../testports.properties
 io.openliberty.checkpoint.dump.threads=true
 com.ibm.ws.logging.trace.specification=*=info:checkpoint=all
-io.openliberty.checkpoint.allowed.features=timedexit-1.0,concurrent-1.0,componenttest-1.0,concurrent-2.0,componenttest-2.0
+io.openliberty.checkpoint.allowed.features=concurrent-1.0,concurrent-2.0
```

Comment:
@dazavala is concurrent needed here because it became part of EE 10 web profile?  Not sure why these features would not have the header to enable checkpoint.

Reply:
concurrent-2.0 has instant on enabled. May be I should just enable cocurrent-1.0 as well.

Reply:
Enabled instant on for concurrent-1.0 and updated the bootstrap properties in FAT.